docs/start_macOS_iOS.md

```diff
@@ -0,0 +1,61 @@
+# 1DS C++ SDK iOS/macOS Podspec Onboarding
+
+This tutorial guides you through the process of integrating the [1DS SDK](https://github.com/microsoft/cpp_client_telemetry) into your existing iOS and macOS app. Here we consume the [Msblox Podspec]() and published [Obj-C Wrappers](https://github.com/microsoft/cpp_client_telemetry/tree/master/wrappers/obj-c). If you want to consume the C++ Library directly, please follow the tutorials at [Getting started - iOS](https://github.com/microsoft/cpp_client_telemetry/blob/master/docs/cpp-start-ios.md) and [Getting Started - Mac OS X](https://github.com/microsoft/cpp_client_telemetry/blob/master/docs/cpp-start-macosx.md)
+
+## Add OneDsCppSdk Cocoapod to the Podfile
+
+The first version is `3.5.67`.
+
+```
+pod 'OneDsCppSdk’, '3.5.67'

```

Comment:
Do we need some podspec file added here, or in the `/build/` directory as a reference?

Reply:
We need to add a source in the Podfile
```
source 'https://msblox.visualstudio.com/DefaultCollection/_git/CocoaPods'
```

I didn't add it since you asked not to mention MSBLOX

Reply:
I'm thinking of adding a "template" that refers to something like _YOUR-PUBLISHING-URL_ or _YOUR-NUGET-ARTIFACTS-URL_ . I am not familiar enough with Cocoapods, but is it possible to use either some variables or environment variables in the spec? Then we can show our generic template here, without mentioning concrete artifacts storage URL.

Reply:
You can probably add a spec file that uses environment variable:
https://stackoverflow.com/questions/58795893/possibility-of-adding-environment-variables-on-podfile

Something like:

`source 'https://' + ENV['COCOAPOD_SOURCE_URL']`

in the example Podfile should do the trick, where in the README here - you'd say that environment variable `COCOAPOD_SOURCE_URL` can be used to specify the artifacts / Cocoapod repository. Folks familiar with the matter can adjust it accordingly.. OR we can just say the users specify the source URL as env var prior to running the `pod` command.

Reply:
i.e.

```console
export COCOAPOD_SOURCE_URL=https://msblox.visualstudio.com/DefaultCollection/_git/CocoaPods
pod ...
```